src/components/nav/sidebarItem.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { FC } from 'react'
+import * as Sidebar from './style'
+import { SidebarLink } from './sidebarLink'
+import { Icon } from 'react-icons-kit'
+
+const genres = [
```

Comment:
I suggest putting this in a constants file.

Reply:
Done ✅